music/models.py

```diff
@@ -0,0 +1,123 @@
+from django.db   import models
+
+class AlbumType(models.Model):
+    name = models.CharField(max_length=10)
+    class Meta:
+        db_table = 'album_types'
```

Comment:
```suggestion
class AlbumType(models.Model):
    name = models.CharField(max_length=10)

    class Meta:
        db_table = 'album_types'
```
모든 클래스에 줄이 붙어있네요. 모두 수정 부탁드립니다!

Reply:
반영했습니다!